.github/checkgroup.yml

```diff
@@ -41,6 +41,25 @@ subprojects:
       - "pl-cpu (windows-2022, 3.10, latest, stable)"
       - "pl-cpu (windows-2022, 3.7, latest, stable)"
       - "pl-cpu (windows-2022, 3.7, oldest, stable)"
+      - "make-doctest (pytorch)"
+      - "make-html (pytorch)"
+      - "mypy"
+      - "PR Gatekeeper (pytorch)"
+      - "pytorch-lightning (GPUs)"
+      - "pytorch-lightning (HPUs)"
+      - "pytorch-lightning (IPUs)"
+      - "pl-slow (macOS-11, 3.7, 1.11)"
+      - "pl-slow (ubuntu-20.04, 3.7, 1.11)"
+      - "pl-slow (windows-2022, 3.7, 1.11)"
+      - "test-on-tpus"
+
+  - id: "lightning_lite"
+    paths:
+      - "requirements/lite/**"
+      - "src/lightning_lite/**"
+      - "tests/tests_lite/**"
+      - "setup.cfg"  # includes pytest config
+    checks:
```

Comment:
I think there shall be also all major Pl tests as PL is now depending on Lite

Reply:
True. I'll do that in a follow-up because that would require updating the path filters for the PL jobs and lite is still WIP